Examples/src/DummyTrkGeoToGdmlDumpSvc.h

```diff
@@ -0,0 +1,38 @@
+#ifndef EXAMPLES_TRKGEOTOGDMLDUMPSVC_H
```

Comment:
This used the Acts example detector for debug purposes. Since the geometry plugin is working now, there is probably little use. I'll remove it.